examples/lock-app/lock-common/lock-app.zap

```diff
@@ -5436,6 +5386,21 @@
               "maxInterval": 65534,
               "reportableChange": 0
             },
+            {
+              "name": "AttributeList",
+              "code": 65531,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
```

Comment:
Why including this?   We're generally not doing that for now to avoid rodata bloat.

Reply:
Will be removed.